views/confirm-login.hbs

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Confirm GitHub Sign in</title>
+  <link rel="stylesheet" href="/static/css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,700,900" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700" rel="stylesheet">
+  <link rel="shortcut icon" href="/static/img/favicon.ico" />
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-3769691-98"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+
+    gtag('config', 'UA-3769691-98');
+  </script>
+</head>
+
+<body>
+
+  <div class="text-center">
+    <div class="container-lg p-responsive py-6 py-lg-10">
+      <img src="/static/img/githubplusslack.svg" class="mb-3 mb-sm-5 col-6 col-sm-12" style="max-width: 300px;">
+      <p>Finish connecting your GitHub account to your slack account on </p>
```

Comment:
Should Slack be capitalized here? Not sure what their branding guidlines are, but since we capitalized GitHub I thought it should be.

```suggestion
      <p>Finish connecting your GitHub account to your Slack account on </p>
```